browser/resources/settings/shortcuts_page/components/ConfigureShortcut.tsx

```diff
@@ -119,10 +98,10 @@ export default function ConfigureShortcut(props: {
 
       // Enter on it's own is used to accept the current combination, if the
       // current keys are a valid shortcut.
-      if (e.code === "Enter" && !hasModifier && maxKeys.current.isValid()) {
+      if (e.code === "Enter" && !hasModifier && current.isValid) {
         props.onChange({
-          codes: keysToString(maxKeys.current.codes),
-          keys: keysToString(maxKeys.current.keys)
+          codes: keysToString(current.codes),
+          keys: keysToString([])
```

Comment:
optional or nvm: maybe it should be like below as we send keys instead?
but still confusing which one is correct here `codes` or `keys` :) 
```suggestion
          keys: keysToString(current.codes)
```

Reply:
Good catch - the keys are displayed until we get a response from the browser process - I'd removed them while I was testing. Added back!